src/NServiceBus.Msmq.AcceptanceTests/When_worker_sends_a_message_for_delayed_retry.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using AcceptanceTesting.Support;
+    using Config;
+    using Features;
+    using NServiceBus.Routing.Legacy;
+    using NUnit.Framework;
+    using ObjectBuilder;
+    using Transport;
+
+    public class When_worker_sends_a_message_for_delayed_retry : NServiceBusAcceptanceTest
```

Comment:
I misread @danielmarbach PR which currently still does send a ready message when a message is sent to SLR. As discussed yesterday, this currently doesn't work when using local TM and the local input queue as the timeout destination address.